crates/extensions/c8y_mapper_ext/src/actor.rs

```diff
@@ -246,15 +160,6 @@ impl C8yMapperActor {
     }
 }
 
-/// Check if a message is an entity registration message.
-fn is_entity_register_message(message: &Message) -> bool {
```

Comment:
This method has been moved into the Converter.

This will have to be updated after https://github.com/thin-edge/thin-edge.io/pull/2164